### ISSUE TITLE
Signifying duplicate usernames

### ISSUE THREAD

**Issue by jgaylor** [TryQuiet/quiet#2685]
Update the text to reflect this change. <img width="838" alt="Image" src="https://github.com/user-attachments/assets/212f943f-18d3-443c-8c0c-06a71e8c7838" />

**Comment by jgaylor** [comment]
@holmesworcester Before I update all the components I want to confirm this is what you had in mind: I mocked it up [here](https://www.figma.com/design/CkkTkVhpCh2lPpF1mRd1vF/username?node-id=2967-9370&t=oMdgxbtMnD2hZHcy-1):

**Comment by holmesworcester** [comment]
"Same name" isn't clear enough, I don't think. I was wrong about updating this everywhere and thanks for checking!

I think we can keep using the "duplicate" label *and* use the new text when we have more space.

![Image](https://github.com/user-attachments/assets/38bc4786-2f8a-4620-9fa8-4c30f7700efd)

**Comment by holmesworcester** [comment]
@jgaylor can you post a link here to whatever warning we're showing the user whose name is a duplicate?

**Comment by jgaylor** [comment]
@holmesworcester I don't think we did anything specific for when 'you' are a duplicate and you're viewing yourself—However, reviewing the designs, 'duplicate' as shown to others users would also be shown next to your own name in all the same places. Perhaps we alter the copy of the red banner if you are also included, but you'd be seeing it by your name anyway. Is that what you meant?

**Comment by holmesworcester** [comment]
Yeah. Though in this case we would want to prompt the users (especially the new user) to change their name.

**Comment by jgaylor** [comment]
@holmesworcester added the mockup to include a ['change username' action](https://www.figma.com/design/CkkTkVhpCh2lPpF1mRd1vF/username?node-id=2909-27390&t=bnEJ4tj4AhDKDqUf-1) when it's you.

**Comment by jgaylor** [comment]
Left a figma comment and a new round of mockups addressing your last round of comments.

**Comment by jgaylor** [comment]
Adding here for reference:

### Reviewing members  

#### Admin actions in a public channel  

- **Viewing duplicate members that are NOT you**  
  - **Action:** Suspend  
  - **Decision helper copy:**  
    - If common private channels (PC) exist: `"In <PC1>, <PC2>, <PC3>, and X more"`  
    - Else: `"No private channels in common"`  

- **Viewing duplicate members that ARE you**  
  - **Action:** Change username  
  - **Helper copy:** N/A  

#### Non-admin actions in a public channel  

- **Viewing duplicate members that are NOT you**  
  - **Action:** Block  
  - **Decision helper copy:**  
    - If common private channels (PC) exist: `"In <PC1>, <PC2>, <PC3>, and X more"`  
    - Else: `"No private channels in common"`  

- **Viewing duplicate members that ARE you**  
  - **Action:** Change username  
  - **Helper copy:** N/A

**Comment by jgaylor** [comment]
@holmesworcester I cleaned up this [Figma Section](https://www.figma.com/design/CkkTkVhpCh2lPpF1mRd1vF/username?node-id=3049-12689&t=syekb6bhJlkDmeOJ-1) based on your last round of comments. Let me know if this looks good to you. I'll update the various components to support these things once we're good.

**Comment by holmesworcester** [comment]
Left one last note about where to display the warning, but all of this looks good to me.

**Comment by jgaylor** [comment]
I made the updates to the warning.

(Edited because I was in the wrong ticket)

**Comment by jgaylor** [comment]
Updated the components in the library. Let me know if there's anything else on this one.
